guides/v2.1/cloud/cdn/fastly-vcl-badreferer.md

```diff
@@ -11,57 +11,104 @@ functional_areas:
   - Setup
 ---
 
-You may want to create a VCL snippet that runs before all other modules to block bad referring websites from accessing your site. To block these sites with a 403 Forbidden error through Fastly, create a VCL snippet to use with an Edge Dictionary of domains to block.
+You can block referrer spam from your {{ site.var.data.ece }} sites using a Fastly Edge Dictionary with custom VCL snippets. The following example shows how to configure Fastly to check the domain name on incoming requests and return a `403 Forbidden` error if the request comes from a domain known for sending bad traffic to your site.
 
-You must have the following information to complete this VCL code snippet:
+{: .bs-callout .bs-callout-info}
+We recommend adding custom VCL configurations to a Staging environment where you can test them before runnng them against the Production environment.
 
-* Create an Edge Dictionary in your environments
-* Account access and URL to the Magento Admin for the Staging or Production environment
+**Prerequisites**
 
-{:.bs-callout .bs-callout-info}
-This information is just the code portion for setting up your VCL. Use this information with [Custom Fastly VCL snippets]({{ page.baseurl }}/cloud/cdn/cloud-vcl-custom-snippets.html).
+-  Configure your {{ site.var.data.ece }} environment for Fastly services. See [Set up Fastly]({{ page.baseurl }}/cloud/cdn/configure-fastly.html). 
 
-## Create an Edge Dictionary {#edge-dictionary}
+-  Get credentials to access the Admin UI for your {{ site.data.var.ece }} environment.
 
-Edge Dictionaries create key-value pairs for running against your VCL snippet. For example, you may want to build a dictionary of URLs to redirect to a Wordpress backend. You may only want to create the edge dictionary in your Production environment. You can also create it in Staging for testing if needed.
+-  Review your site log for traffic spamming your site and note the domain names that you need to block.
 
-1. Log in to the Magento Admin.
-2. Navigate to **Stores** > **Configuration** > **Advanced** > **System** > **Fastly Configuration**.
-3. Expand the **Edge dictionaries** section.
-4. Click **Add container**. You need to create a container to hold up to 1,000 key-value pairs.
-5. On the container, enter a Dictionary name. For this example, use the name `referer_blocklist`.
-6. Select the checkbox for **Activate after the change** if you want to the dictionary after creating or editing the container.
-7. Add key-value pairs in the new dictionary. For this example, enter the URLs for your blog that should be redirected to your Wordpress backend. Enter a value of 1.
+
+## Create a referrer block list
+
+Edge Dictionaries create key-value pairs accessible to VCL functions during VCL snippet processing. In this example, you create an edge dictionary that provides the list of website referrers that you want to block.
```

Comment:
To avoid wanting things, maybe you can remove "that you want"

-> ...provides a list of website referrers to block.
Just a thought.